src/compiler/crystal/macros.cr

```diff
@@ -34,12 +34,24 @@ module Crystal::Macros
   def flag?(name) : BoolLiteral
   end
 
-  # Prints an AST node at compile-time. Useful for debugging macros.
-  def puts(expression) : Nop
+  # Prints AST nodes at compile-time. Useful for debugging macros.
+  def puts(*expressions) : Nop
   end
 
   # Same as `puts`.
```

Comment:
`# :ditto:`?

Reply:
I'm not sure that's really worth it for three short words...

Reply:
umm, u r right.

![image](https://user-images.githubusercontent.com/988/43552987-435741ca-95ed-11e8-9744-b3084b75e1e2.png)